### DISCUSSION TITLE
AFE op amp choices

### DISCUSSION THREAD

**Discussion by Faqarini** [fhdm-dev/scoppy#148] [Q&A]
Was the mcp6002 still a good choice? Or is there a better recommendation I can look for? 

How about the mcp602? I saw that it has bigger bandwidth and higher slew rate.

**Comment by fhdm-dev** [comment]
Hi,
I don't think the MCP6002 is still a good choice now that the sampling rate can be increased to 2MS/s.
I would choose something with a higher bandwidth and slew rate than the MCP602. 

[Jason-BCIT] seems to have had some success with the TLV9064. See:
[https://github.com/fhdm-dev/scoppy/discussions/139](https://github.com/fhdm-dev/scoppy/discussions/139)

**Reply by Faqarini** [reply]
Thanks for the recommendation, but unfortunately I was not able to find TLV9064 on our region. 
On the other hand, I have found the MCP6022 smd ic. I hoped it is genuine or a good enough clone. I will try making picoLABO's circuit in the meantime.

Btw... I have experimented with the RP2040-Zero board and it works. What a little board, I hoped I can also make a small AFE with through hole components.

**Reply by Jason-BCIT** [reply]
Sorry for not replying sooner, but yes, the TLV9064 is working very well using the Pico’s 0-3.3V supply as the power source. On a previous design I used an MCP6024, but that used a 0-5V supply (from the USB) and that gave enough voltage range to avoid much of the signal loss at the high and low ends of the range. I think older opamps that aren’t true rail to rail on both inputs and outputs might work okay if the board is powered by a 6V or 9V battery. The signal can be processed at a higher voltage and then fed to the Pico through a voltage divider.

I’m on vacation for the next little while, but hope to post some screenshots of the TLV’s performance as I prepare my lessons for my students later in August… but I’m very happy with it! When reading a 200kHz square wave I do see some slope on the edges, but will have to see if that is a slew rate issue, or simply a sampling rate issue. Either way it is well within expectations for this project! Thank you for suggesting the TLV… that saved several iterations!
